code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -29,6 +29,10 @@
 /obj/item/reagent_containers/hypospray/borghypo/rescue
 	reagent_ids = list(/decl/reagent/tricordrazine, /decl/reagent/dexalin, /decl/reagent/inaprovaline, /decl/reagent/dylovene, /decl/reagent/perconol, /decl/reagent/mortaphenyl, /decl/reagent/adrenaline, /decl/reagent/coagzolug)
 
+/obj/item/reagent_containers/hypospray/borghypo/hacked
+	reagent_ids = list(/decl/reagent/toxin/zombiepowder, /decl/reagent/toxin/cyanide, /decl/reagent/polysomnine, /decl/reagent/toxin/panotoxin, /decl/reagent/toxin/berserk, /decl/reagent/wulumunusha)
```

Comment:
i don't really like zombie powder nor cyanide in a unremovable, infinite hypospray.

Reply:
With a delayed recharge time, and an older hypospray time which has a fairly noticeable delay for injection. Not too useful in combat, it’s a tool for assassination and kidnapping.

Reply:
!review